app/services/rapid_ready_for_decision/constants.rb

```diff
@@ -14,7 +14,33 @@ class Constants
         code: 6602,
         label: 'asthma',
         flipper_name: 'asthma',
-        sidekiq_job: 'RapidReadyForDecision::Form526AsthmaJob'
+        sidekiq_job: 'RapidReadyForDecision::Form526AsthmaJob',
+        keywords: %w[
+          Aerochamber
+          Albuterol
+          Beclomethasone
+          Benralizumab
+          Budesonide
+          Ciclesonide
+          Fluticasone
+          Levalbuterol
+          Mepolizumab
+          Methylprednisolone
+          Mometasone
+          Montelukast
+          Omalizumab
+          Prednisone
+          Reslizumab
+          Salmeterol
+          Theophylline
+          Zafirlukast
+          Zileuton
+          Asthma
+          Breath
+          Inhale
```

Comment:
I believe this was intentionally shortened to capture other word forms like "inhalation"
```suggestion
          Inhal
```